frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.all.flavors.ts

```diff
@@ -0,0 +1,29 @@
+import { withResource } from '@console/shared/src/test-utils/utils';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { V2V_VM_IMPORT_TIMEOUT, VM_STATUS } from '../tests/utils/consts';
+import { VirtualMachine } from '../tests/models/virtualMachine';
+import { flavorConfigs } from './v2v.configs';
+
+describe('Kubevirt create VMs using wizard, all flavors and profiles', () => {
+  const leakedResources = new Set<string>();
+//   const notCleanEnv = true;
```

Comment:
Please remove the commented code

Reply:
Done

Reply:
Well, I can still see that the comments are here, can you update the PR?

Reply:
All changes pushed